src/pages/en/guides/markdown-content.md

```diff
@@ -312,6 +312,29 @@ import {Content as PromoBanner} from '../components/promoBanner.md';
 <PromoBanner />
 ```
 
+When used with `Astro.glob()`, you can simply pass the `<Content/>` component from the _Exported Properties_ of each returned markdown file. This will then render out the Markdown file and its contents in its entirety into your template. 
+
+```astro
+---
+export async function getStaticPaths() {
+  const posts = await Astro.glob('../posts/**/*.md')
+
+  return posts.map(post => ({
+    params: { 
+      slug: post.frontmatter.slug 
+    },
+    props: {
+      post
+    },
+  }))
+}
+
+const { post:{ Content } } = Astro.props
```

Comment:
Another nit, but I’m not a huge fan of multiple destructuring. Do you think this might be easier to read/understand?

```suggestion
const { Content } = Astro.props.post
```